contracts/DePayRouterV1PaymentEvent02.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.6 <0.9.0;
+pragma abicoder v2;
+
+contract DePayRouterV1PaymentEvent02 {
+
+  // The payment event.
+  event Payment(
+    address indexed sender,
+    address payable indexed receiver,
+    uint256 indexed amount,
```

Comment:
Do the amount is exact amount?.

Reply:
This is gonna be the exact amount as configured in the widget and payed to the receiver. See payment plugin: https://github.com/DePayFi/depay-evm-router/blob/master/contracts/DePayRouterV1Payment01.sol#L32